### PR TITLE
[MIG] account_bank_statement_import_paypal: Migration to 12.0

### PR DESCRIPTION
**Pull request by cubells**
Standard migration

cc @Tecnativa

### CODE REVIEW

File: account_bank_statement_import_paypal/data/paypal_map_data.xml [hunk 1]
```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" ?>
+<odoo noupdate="1">
+
+    <record id="paypal_map" model="account.bank.statement.import.paypal.map">
+        <field name="name">Paypal Monthly Statement</field>
```
Comment (pedrobaeza) [inline account_bank_statement_import_paypal/data/paypal_map_data.xml]: Add ` (English)` at the end for recognizing that is in that language

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
I'm missing here Spanish paypal map configuration, and check also runbot warnings:
```
2019-04-24 10:37:08,482 161 WARNING openerp_test odoo.models: The model account.bank.statement.import.paypal.map has no _description
2019-04-24 10:37:08,483 161 WARNING openerp_test odoo.models: The model account.bank.statement.import.paypal.map.line has no _description
2019-04-24 10:37:08,483 161 WARNING openerp_test odoo.models: The model wizard.paypal.map.create has no _description
```

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]
I would like to see separated the IMP commit of v11 and the MIG commit to v12. Now they are fused :S

**Comment by cubells** [comment]
> I'm missing here Spanish paypal map configuration, and ...

Do you want I add an spanish paypal map in data?

**Comment by pedrobaeza** [comment]
> Do you want I add an spanish paypal map in data?

Yes, as it's needed for our customer, and it's the same valid predefined data as English one.

**Comment by cubells** [comment]
This is WIP please.

**Comment by pedrobaeza** [comment]
@victormmtorres please re-take this and include #211 here

**Comment by victor-ditay** [comment]
cc @pedrobaeza changes done

File: account_bank_statement_import_paypal/models/account_bank_statement_import_paypal_map.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
+# Copyright 2019 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, api
+
+
+class AccountBankStatementImportPaypalMap(models.Model):
+    _name = 'account.bank.statement.import.paypal.map'
+
```
Comment (MiquelRForgeFlow) [inline account_bank_statement_import_paypal/models/account_bank_statement_import_paypal_map.py:10]: add `_description`

File: account_bank_statement_import_paypal/models/account_bank_statement_import_paypal_map.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                      'none': '',
+                      }
+        return (separators[self.float_thousands_sep],
+                separators[self.float_decimal_sep])
+
+
+class AccountBankStatementImportPaypalMapLIne(models.Model):
+    _name = 'account.bank.statement.import.paypal.map.line'
+    _order = "sequence asc, id asc"
+
```
Comment (MiquelRForgeFlow) [inline account_bank_statement_import_paypal/models/account_bank_statement_import_paypal_map.py:71]: add `_description`

File: account_bank_statement_import_paypal/wizards/create_map_lines_from_file.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+import csv
+import base64
+from odoo import api, fields, models
+from io import StringIO
+
+
+class WizardPaypalMapCreate(models.TransientModel):
+    _name = 'wizard.paypal.map.create'
+
```
Comment (MiquelRForgeFlow) [inline account_bank_statement_import_paypal/wizards/create_map_lines_from_file.py:13]: add `_description`

File: account_bank_statement_import_paypal/models/account_bank_statement_import_paypal_map.py [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+    def _get_separators(self):
+        separators = {'dot': '.',
+                      'comma': ',',
+                      'none': '',
+                      }
+        return (separators[self.float_thousands_sep],
+                separators[self.float_decimal_sep])
+
+
+class AccountBankStatementImportPaypalMapLIne(models.Model):
```
Comment (MiquelRForgeFlow) [inline account_bank_statement_import_paypal/models/account_bank_statement_import_paypal_map.py]: ...LIne -> ...Line

**Comment by victor-ditay** [comment]
Changes done @mreficent

**Comment by MiquelRForgeFlow** [comment]
... and runbot is being stupid.

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**